packages/strapi-design-system/src/ToggleCheckbox/ToggleCheckbox.js

```diff
@@ -44,6 +45,14 @@ const ValueBox = styled(Flex).attrs({
   position: relative;
   user-select: none;
   z-index: 2;
+  flex: 1 1 50%;
+  /**
+    We declare the defined value because we want the height of the input when 
+    the values are in a row to be 40px. But defining a height on the label
+    would break the input when it wraps.
+  */
+  padding-top: ${({ size }) => `${size === 'S' ? '2px' : '6px'}`};
+  padding-bottom: ${({ size }) => `${size === 'S' ? '2px' : '6px'}`};
```

Comment:
I think @ronronscelestes will comment that these should be defined as props on `ToggleCheckboxWrapper`, because `Box` provides `paddingTop` and `paddingBottom` already.

Reply:
agree, I'm also worried about `6px` since it's not a theme spacing value but I guess we don't really have a choice here 🤔

Reply:
> I'm also worried about 6px

What about adding a code-comment to reason about these magic numbers?

Reply:
you mean like this:

```js
  /**
    We declare the defined value because we want the height of the input when 
    the values are in a row to be 40px. But defining a height on the label
    would break the input when it wraps.
  */
```

Which already is there, but maybe it's not clear enough?

Reply:
Ah, sorry - forgot about it, because the comment only refers to two lines. The comment works for me.